docs/src/modules/utils/generateMarkdown.js

```diff
@@ -94,6 +94,10 @@ function resolveType(type) {
     return type.elements.map((t) => resolveType(t)).join(' \\| ');
   }
 
+  if (type.type === 'StringLiteralType') {
```

Comment:
Allows for correct markdown generation for string unions in jsdoc.
https://github.com/mui-org/material-ui/pull/20609/commits/0fdc32a9a98a742a1f69696f8454fa3f2acf2d7f#r410713161

Reply:
I will create a separate pull request for this.